import-api/mastr_storage.json

```diff
@@ -0,0 +1,162 @@
+{"name": "supply.bnetza_mastr",
+"title": "Bundesnetzagentur - Marktstammdatenregister - Stromerzeuger",
```

Comment:
Stromspeicher

Reply:
@Izzet91 did it in 8fd50a32a5e540c143e9de5258dd15051279de31